subiquity/server/controllers/install.py

```diff
@@ -101,20 +114,37 @@ def tpath(self, *path):
     def log_event(self, event):
         self.tb_extractor.feed(event['MESSAGE'])
 
-    def write_config(self):
+    def write_config(self, config_file: Path, config: Any) -> None:
+        """ Create a YAML file that represents the curtin install configuration
+        specified.  """
+        config_file.parent.mkdir(parents=True, exist_ok=True)
+        generate_config_yaml(str(config_file), config)
+
+    def acquire_generic_config(self, step: CurtinInstallStep,
+                               resume_data_file: Path) -> Dict[Any, Any]:
```

Comment:
Pedantry in the extreme but I think all the keys in a curtin config are strings :-)

Reply:
Correct :)